### PR TITLE
Distribution License: Don't cover internal-use work (close #38)

### LINKED ISSUE #38: Should Internal Business Use limit changes to internal-use changes?
IB currently reads:

> It is a permitted purpose to use the software for the internal business operations of your company.

- [x] "Use" may too specific, for the same reasons it was too specific in Noncommerical Organizations.

- [ ] If a particular form also includes distribution and changes licenses, should IB more clearly limit creation of changes and new work to internal-use projects?

### PR DESCRIPTION
**Pull request by kemitchell**
This PR changes the terms so that when Distribution License, Changes and New Work License, and Internal Business Use all appear in a variant, Distribution License does _not_ allow distribution of changes and new work made for internal use.

@heathermeeker this is really a question about the specification for `Polyform-Internal`. Should it permit distributing internal-use changes or not?

### CODE REVIEW

File: polyform.md [hunk 1]
```diff
@@ -14,7 +14,9 @@ The licensor grants you a copyright license for the software, to do everything y
 
 ## DL: Distribution License
 
-The licensor grants you an additional copyright license to distribute copies of the software, and if these terms grant you an additional license to make changes and new works, to distribute your changes and new works, as well.  You must ensure that anyone who gets a copy of any part of the software from you also gets the text of these terms or the URL for its text above.
+The licensor grants you an additional copyright license to distribute copies of the software.  If these terms grant you an additional license to make changes and new works, your license to distribute covers distributing your changes and new works, as well.  However, if these terms allow you to make changes and new works for internal business operations, your license to distribute does not cover distributing those changes and new works.
```
Comment (karianna) [inline polyform.md]: Don't need the comma in `changes and new works, as well`

File: polyform.md [hunk 2]
```diff
@@ -14,7 +14,9 @@ The licensor grants you a copyright license for the software, to do everything y
...
+
+You must ensure that anyone who gets a copy of any part of the software from you also gets the text of these terms or the URL for its text above.
```
Comment (karianna) [inline polyform.md]: Don't need the word `above`?

Comment (kemitchell) [reply polyform.md]: I don't like using "above", but we want to be clear that the URL folks can use is the URL at the top of the license.

Perhaps we put "Official URL:" or "Permalink:" before the URL at the top, and then allow using the "official URL" in this section?

File: polyform.md (hunk 1 above)
Comment (kemitchell) [reply polyform.md]: Struck "as well" altogether in e38d80d78f3c8ebbc84e798018293a1705e7b5fd

File: polyform.md (hunk 2 above)
Comment (karianna) [reply polyform.md]: SGTM

**Pull request merged**